frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx

```diff
@@ -355,15 +354,22 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
 
           <div className="page-body">
             <div className="container-xl">
+              {isLoadingGroup && (
```

Comment:
Spinner is not used as page loader
Please try something like this 
https://github.com/ToolJet/ToolJet/blob/develop/frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx#L447